app/src/main/java/bg/zahov/app/ui/measures/info/MeasurementInfoFragment.kt

```diff
@@ -9,125 +8,53 @@ import android.view.MenuItem
 import android.view.View
 import android.view.ViewGroup
 import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
 import androidx.core.view.MenuProvider
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
-import androidx.lifecycle.map
 import androidx.navigation.fragment.findNavController
-import bg.zahov.app.data.model.state.MeasurementInformationUiMapper
 import bg.zahov.app.hideBottomNav
 import bg.zahov.app.setToolBarTitle
 import bg.zahov.app.showTopBar
 import bg.zahov.app.ui.measures.MeasuresFragment.Companion.MEASUREMENT_ARGS
-import bg.zahov.app.ui.measures.info.input.MeasurementInputFragment
-import bg.zahov.app.util.MonthValueFormatter
-import bg.zahov.app.util.RightAxisValueFormatter
 import bg.zahov.fitness.app.R
-import bg.zahov.fitness.app.databinding.FragmentMeasurementInformationBinding
-import com.github.mikephil.charting.components.XAxis
-import com.github.mikephil.charting.data.LineData
-import com.github.mikephil.charting.data.LineDataSet
-import java.time.LocalDate
 
 class MeasurementInfoFragment : Fragment() {
-    private var _binding: FragmentMeasurementInformationBinding? = null
-    private val binding
-        get() = requireNotNull(_binding)
-    private val measurementInfoViewModel: MeasurementInfoViewModel by viewModels()
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View {
-        _binding = FragmentMeasurementInformationBinding.inflate(inflater, container, false)
-        requireActivity().showTopBar()
-        requireActivity().hideBottomNav()
-        return binding.root
-    }
-
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
+        val title = arguments?.getString(MEASUREMENT_ARGS) ?: "Measurement"
```

Comment:
Default value should be extracted string.

Reply:
i need to get context for extracted string but context is also nullable

Reply:
probably will be changed how it works when we no longer use fragments

Reply:
I think you can set it to the argument value first, then fall back to an extracted resource, and if context is null you can use empty string.  Also `onViewCreated()` shouldn't have null context but we access it optionally to be extra safe.

Reply:
done